### DISCUSSION TITLE
Vercel - Supabase intergration fail

### DISCUSSION THREAD

**Discussion by brighton-ifaya** [vercel/community#3194] [Bug Report]
### Page to Investigate

https://vercel.com/integrations/supabase

### Steps to Reproduce

I have been trying for the past one hour to intergrate supabase into vercel. I have a project already setup on vercel and a database already setup on supabase. I, however much I retry, keep getting redirected to supabase's home page when I click the "intergrate button on vercel". Now I don't even know who to specifically seek help from the two so I am going to try both sides and see where the users are experiencing similar problems. If you have experienced such, would you mind telling me the steps you took to resolve?

**Comment by amyegan** [comment]
I'm sorry to hear you're running into problems, @shifu-brighton. The Supabase integration is not developed or maintained by Vercel, so I recommend reaching out to [Supabase Support](https://supabase.com/support) for help with this 🙇

**Comment by ThomasWT** [comment]
@shifu-brighton It seems to be working now. I had the same issue yesterday.

**Reply by leerob** [reply]
We got in touch with the Supabase team and they quickly fixed things 🙏

**Reply by brighton-ifaya** [reply]
Yes I can confirm it is working now. 👌🏾
